libs/vkd3d/state.c

```diff
@@ -2474,6 +2488,9 @@ static HRESULT vkd3d_late_compile_shader_stages(struct d3d12_pipeline_state *sta
             {
                 break;
             }
+
+            // Vulkan requires identifier to be not present when shader module is specified.
+            vkd3d_shader_stage_remove_identifier_pnext(&graphics->stages[i]);
```

Comment:
This is not true, identifierSize must be 0 instead.

Reply:
You're right. Should I just update the comment or also modify the helper to set size to 0?

Reply:
Just setting identifierSize to 0 is problematic too actually, because that identifier might be read by the disk cache threads. Let's go with pNext removal instead.

Reply:
Can update the comment to reflect this.